IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouseTemplate.mo

```diff
@@ -0,0 +1,103 @@
+within IBPSA.Examples.Tutorial.SimpleHouse;
+model SimpleHouseTemplate
```

Comment:
Why is this called template and not partial, 0, or inputs? The template is not used any more.

Reply:
I agree that `SimpleHouse0` would be better, as it is not really a template.

Reply:
I renamed `SimpleHouseTemplate` to `SimpleHouse0`